app/code/Magento/Downloadable/view/frontend/templates/email/order/items/order/downloadable.phtml

```diff
@@ -5,42 +5,46 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block \Magento\Downloadable\Block\Sales\Order\Email\Items\Order\Downloadable */ ?>
+<?php
```

Comment:
Please remove not needed closing and opening PHP tags

Reply:
fixed